config/log4j2.properties

```diff
@@ -16,63 +16,32 @@
 # limitations under the License.
 ################################################################################
 
-property.file_path = ${sys:seatunnel.logs.path:-/tmp/seatunnel/logs}
-property.file_name = ${sys:seatunnel.logs.file_name:-seatunnel}
-property.file_split_size = 100MB
-property.file_count = 100
-property.file_ttl = 7d
+appender.console.type=Console
```

Comment:
Why change this file?

Reply:
Because I found the file log can not work in seatunnel cluster.